pom.xml

```diff
@@ -26,10 +26,10 @@
         <version>1.0.5</version>
     </parent>
 
-    <groupId>org.eclipse.ee4j.servlet</groupId>
+    <groupId>jakarta.servlet</groupId>
     <artifactId>servlet-parent</artifactId>
     <packaging>pom</packaging>
-    <version>5.0-SNAPSHOT</version>
```

Comment:
I think this is incorrect, I believe the document should not support micro versions, as per a comment from a previous PR.